testsuite/elytron/src/test/java/org/wildfly/test/security/common/elytron/SimpleServerSslContext.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.test.security.common.elytron;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.as.test.integration.management.util.CLIWrapper;
+
+/**
+ * Elytron server-ssl-context configuration implementation.
+ *
+ * @author Josef Cacek
+ */
+public class SimpleServerSslContext extends AbstractConfigurableElement {
+
+    private final String keyManager;
+    private final String trustManager;
+    private final String securityDomain;
+    private final String[] protocols;
+    private final boolean needClientAuth;
+    private final Boolean authenticationOptional;
+
+    private SimpleServerSslContext(Builder builder) {
+        super(builder);
+        this.keyManager = builder.keyManager;
+        this.trustManager = builder.trustManager;
+        this.securityDomain = builder.securityDomain;
+        this.protocols = builder.protocols;
+        this.needClientAuth = builder.needClientAuth;
+        this.authenticationOptional = builder.authenticationOptional;
+    }
+
+    @Override
+    public void create(ModelControllerClient client, CLIWrapper cli) throws Exception {
+        // /subsystem=elytron/server-ssl-context=twoWaySSC:add(key-manager=twoWayKM,protocols=["TLSv1.2"],
+        // trust-manager=twoWayTM,security-domain=test,need-client-auth=true)
+        StringBuilder sb = new StringBuilder("/subsystem=elytron/server-ssl-context=").append(name).append(":add(");
```

Comment:
I've reused [the implementation from WildFly]( https://github.com/wildfly/wildfly/blob/master/testsuite/shared/src/main/java/org/wildfly/test/security/common/elytron/SimpleServerSslContext.java) here. If it doesn't fit into core, I can refactor it.